src/codeflare_sdk/templates/base-template.yaml

```diff
@@ -8,6 +8,8 @@ metadata:
     orderedinstance: "m4.xlarge_g4dn.xlarge"
 spec:
   priority: 9
+  schedulingSpec:
+    minAvailable: 3
```

Comment:
Just curious, what is the `minAvailable` field for/mean?

Reply:
[Expected number of pods in running and/or completed state. Requeuing is triggered
when the number of running/completed pods is not equal to this value. When not specified,
requeuing is disabled and no check is performed.](https://github.com/project-codeflare/multi-cluster-app-dispatcher/blob/9035cd73d5bd03a1b584b3567255f6af5006e7d6/config/crd/bases/mcad.ibm.com_appwrappers.yaml#LL262C27-L262C27)